recipes/hmat-oss/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.1.2" %}
+{% set variant = "openblas" %}
+
+package:
+  name: hmat-oss
+  version: {{ version }}
+
+source:
+  fn: hmat-oss-{{ version }}.tar.gz
+  url: https://github.com/jeromerobert/hmat-oss/archive/{{ version }}.tar.gz
+  sha1: fe52fa22e413be862bec1b44a2b695a566525138
+  patches:
+    - 7d14b6451d3f63a949fee9c09d349aec6edd193e.patch
```

Comment:
Can we please add a note above here that just says these are commits from the repo?